sql/moz-fx-data-shared-prod/fx_quant_user_research_derived/fxqur_viewpoint_desktop_v1/metadata.yaml

```
@@ -22,6 +22,8 @@ scheduling:
       "{{ var.value.surveygizmo_api_secret }}",
       "--destination_table",
       "moz-fx-data-shared-prod.fx_quant_user_research_derived.fxqur_viewpoint_desktop_v1",
+      "--include_url_variables",
+      True
```

The `True` here is problematic since `arguments` is expected to be a list of strings (plus in case-sensitive YAML booleans are expected to be lowercase).  However, if you take my suggestion from another comment to specify `is_flag=True` for this option then I don't think you'd need to specify the value at all.
```suggestion
      "--include_url_variables"
```
(same goes in the other `metadata.yaml` file)